src/auth/auth.js

```diff
@@ -0,0 +1,50 @@
+// Modules
+import axios from "axios";
+
+const getAADToken = async () => {
```

Comment:
it'd be good to check if we are in localhost, as when working with it locally we get spammed with network requests to /auth/.me that constantly fail and throw errors.

Reply:
would work

```
if (process.env.NODE_ENV === 'production') {
}
```

Reply:
you don't have environment variables when you do build. It is just static HTML, CSS and JS

Reply:
I can put enable-auth as a build option, I will do that (y)

Reply:
Thank you for your input, I will code it in and let you know. 

If it works and feels okay, we can do it the same way on the frontend too if you would like!

Reply:
we could use the REACT_APP_ env